<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SaveXMLButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to XML</string>
    </property>
   </widget>
   <widget class="DrawPad" name="DrawPadWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="HistogramView" name="HistogramViewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RecognizeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recognize</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="DigitRadioBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>What is the digit?</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearDrawButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear draw pad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TeachButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Teach</string>
    </property>
   </widget>
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecognizer">
    <property name="title">
     <string>Recognizer</string>
    </property>
    <widget class="QMenu" name="menuTeach_from_base">
     <property name="title">
      <string>Teach from database</string>
     </property>
     <addaction name="actionBase_directory"/>
     <addaction name="actionFiles"/>
    </widget>
    <widget class="QMenu" name="menuTest_from_base">
     <property name="title">
      <string>Test from database</string>
     </property>
     <addaction name="actionBase_directory_2"/>
     <addaction name="actionFiles_2"/>
    </widget>
    <addaction name="actionLoad"/>
    <addaction name="menuTeach_from_base"/>
    <addaction name="menuTest_from_base"/>
    <addaction name="separator"/>
    <addaction name="actionShow_neurons"/>
    <addaction name="actionErase_neurons"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuRecognizer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load neurons</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionErase_neurons">
   <property name="text">
    <string>Erase neurons</string>
   </property>
  </action>
  <action name="actionShow_neurons">
   <property name="text">
    <string>Show neurons</string>
   </property>
  </action>
  <action name="actionBase_directory">
   <property name="text">
    <string>Base directory</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionBase_directory_2">
   <property name="text">
    <string>Base directory</string>
   </property>
  </action>
  <action name="actionFiles_2">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawPad</class>
   <extends>QWidget</extends>
   <header>drawpad.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistogramView</class>
   <extends>QWidget</extends>
   <header>histogramview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SaveXMLButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
